apps/builder/app/canvas/features/text-editor/text-editor.tsx

```diff
@@ -63,6 +64,27 @@ const AutofocusPlugin = () => {
   return null;
 };
 
+const InterceptBlurPlugin = (props: { onBlur: () => void }) => {
+  const [editor] = useLexicalComposerContext();
+  const onBlur = useEffectEvent(props.onBlur);
+
+  useEffect(() => {
+    const rootElement = editor.getRootElement();
+
+    if (rootElement === null) {
+      return;
+    }
+
+    rootElement.addEventListener("blur", onBlur);
+
+    return () => {
+      rootElement.removeEventListener("blur", onBlur);
+    };
+  }, [editor, onBlur]);
+
+  return null;
```

Comment:
does it still require a null? react takes undefined these days
```suggestion
  return;
```

Reply:
<img width="495" alt="image" src="https://github.com/webstudio-is/webstudio/assets/5077042/bcd2f975-ea8f-4b5e-bc9f-19becfe77241">